src/main/resources/verily_auth0_dev_secret.json

```diff
@@ -0,0 +1,10 @@
+{
+  "installed": {
+    "project_id": "terra-ots0-prod",
```

Comment:
prod?

Reply:
i guess i don't know what this is used for?

Reply:
> The current baseline code is broken (latest release 0.352.0 stable) i am working on a fix via #497 Let me know if this is needed right away, otherwise please hold off on this for the rest of the day. Thanks

just sending this off for review, i will wait to merge.

Reply:
> i guess i don't know what this is used for?

oops this is copied over. removed